api/api.go

```diff
@@ -292,3 +292,28 @@ func (c *Client) Skip(language, slug string) error {
 
 	return errors.New(pe.Error)
 }
+
+// Status sends a request to exercism to fetch the user's
+// completion status for the given language track.
+func (c *Client) Status(language string) (*StatusInfo, error) {
```

Comment:
I guess the one thing that might still want to change is if we want to rename these variables across all the `Client` methods to be `trackId` or something like that to be consistent with the field names in the various types that use that information.